next-docs/pages/templates/typescript/react.mdx

```diff
@@ -1 +1,86 @@
 # React
+
+This template is for developers who use [React](https://react.dev/).
+
+## Onchain code
+
+This template uses [the standard `Counter` onchain code](./contracts).
+
+## Offchain code
+
+The files you are likely to need to change in the offchain code are:
+
+- [`packages/client/src/App.tsx`](https://github.com/latticexyz/mud/blob/main/templates/react/packages/client/src/App.tsx), which controls the information displayed to the user.
+  It is the place where you register handlers that are called when the value in a component (basically, a table) changes.
+- [`packages/client/src/mud/createSystemCalls.ts`](https://github.com/latticexyz/mud/blob/main/templates/react/packages/client/src/mud/createSystemCalls.ts), which is where you write code that performs calls to an onchain system.
+
+### `App.tsx`
+
+```ts
+export const App = () => {
+  const {
+    components: { Counter },
+    systemCalls: { increment },
+  } = useMUD();
+```
+
+Call the setup.
```

Comment:
This hook just returns the state on the MUD context, the setup happens in `index.tsx` (line 12)

Reply:
Thank you.